src/components/Admin/UserPage/ManageUsers.tsx

```diff
@@ -38,10 +47,11 @@ const ManageUsers: () => JSX.Element = () => {
     // Only fetch all user data if user is an admin user
     if (isAdmin) {
       setIsLoading(true);
-      firestore
+
+      // Creates a listener that updates the data on any changes
+      const unsubscribe = firestore
```

Comment:
why is this called unsubscribe? (as opposed to something like "getUsers")

Reply:
The snapshot listener (listening for doc changes) returns a function to "unsubscribe", which is later returned so that when the component is unmounted, the listener is unsubscribed